highcharts-angular/package.json

```diff
@@ -21,7 +21,8 @@
   "peerDependencies": {
     "@angular/common": ">=6.0.0",
     "@angular/core": ">=6.0.0",
-    "highcharts": ">=6.0.0"
+    "highcharts": ">=6.0.0",
+    "typescript": ">=3.8.2"
```

Comment:
Instead of adding an explicit deps on typescript, I think we should increase the requirement of `@angular/common` and `@angular/core` to be `">=9.0.0"`, because this really is what it is all about.